### ISSUE TITLE
Simple even number check using bitwise operators.

### ISSUE THREAD

**Issue by shadowmaster435** [godotengine/godot-proposals#8693]
### Describe the project you are working on

N/A

### Describe the problem or limitation you are having in your project

N/A

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

A convenient method to go with ones such as, "is_inf" or "is_nan" 

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Works with this simple equation 
`(integer^1) & 1`

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Yes

### Is there a reason why this should be core and not an add-on in the asset library?

Its a single method.

**Comment by Calinou** [comment]
`integer % 2 == 0` is the easiest way to do it, even if it's not the fastest. (If you have to ask, it's probably not a bottleneck in your code.)

**Comment by AThousandShips** [comment]
In either case GDScript itself will be a bigger bottleneck than the code, including a library call

**Comment by fire** [comment]
I don't think this should be in Godot Engine, but if you wanted to expose this for your own uses, I would use the existing cryptographic mbedtls's method of large numbers math.

**Comment by nicolasgustafsson** [comment]
`is_even(integer)` reads better to me than `integer % 2 == 0`. According to what I can tell from @Calinou's comment it's also faster. So it seems that it's just a straight upgrade.

What are the arguments as to why it shouldn't be part of core?

**Comment by AThousandShips** [comment]
Faster is relative here, calling `is_even` isn't free either, so there's margins, we can't add a function for every thing that goes slightly faster when it's something that you can easily do and work around 🙂 there's very little need for a slightly faster version of this
